guides/extensions/push-notifications/setup.md

```diff
@@ -0,0 +1,9 @@
+# Setup
+
+To use the `push-notifications` package, you need to have set up Aries Framework JavaScript according to the AFJ repository (See `Getting Started` section in the sidebar).
```

Comment:
Can we add links to the getting started guide here instead of referring to the sidebar?